src/ripple/beast/utility/PropertyStream.h

```diff
@@ -139,9 +133,9 @@ class PropertyStream
     add(signed char value);
     virtual void
     add(unsigned char value);
+#if 0
```

Comment:
Hmmm.  I'd read @nbougalis's suggesting as suggesting the `#if`d out code be removed as well.  But maybe I misunderstood.

Reply:
I thought I removed them, but I guess I missed this. Will fix.

Reply:
Fixed in fcf5f2cf6e [fold] Rm more dead code